### PR TITLE
appleseed.package.py script updated for Qt5 library

### PR DESCRIPTION
**Pull request by przemb**
Hi, 
I have updated script appleseed.package.py file, mentioned in https://github.com/appleseedhq/appleseed/issues/2540

Please verify it.

### CODE REVIEW

File: scripts/appleseed.package.py [hunk 1]
```diff
@@ -507,7 +507,7 @@ def add_python_to_stage(self):
         shutil.copy(os.path.join(self.settings.python_path, "README.txt"), "appleseed/python27")
 
     def copy_qt_framework(self, framework_name):
-        src_filepath = os.path.join(self.settings.qt_runtime_path, framework_name + "4" + ".dll")
+        src_filepath = os.path.join(self.settings.qt_runtime_path, framework_name + "5" + ".dll")
         dst_path = os.path.join("appleseed", "bin")
```
Comment (jdent02) [inline scripts/appleseed.package.py]: Did you actually check that this works?  The name of the file under Qt 4 was QtCore4.dll.  Under Qt5 it’s Qt5Core.dll

**Comment by pjessesco** [comment]
I've working on it, and at least I should copy `QtConcurrent` and `QtWidgets` too. You can check `otool` to see dependencies.

**Comment by Mango-3** [comment]
On Windows the required QT5 dll files are:
Qt5Core.dll
Qt5Gui.dll
Qt5OpenGL.dll
Qt5Widgets.dll

**Comment by przemb** [comment]
Due to problems with building project, which I described on Discord, I am temporarily blocked.
Feel free to take this issue.

**Pull request closed without merging**